test/liblll/EndToEndTest.cpp

```diff
@@ -279,6 +279,26 @@ BOOST_AUTO_TEST_CASE(assembly_codecopy)
 	BOOST_CHECK(callFallback() == encodeArgs(string("abcdef")));
 }
 
+BOOST_AUTO_TEST_CASE(returndatasize_can_be_compiled)
+{
+	char const* sourceCode = R"(
+		(returnlll
```

Comment:
OK, I can guess from the other tests that opcodes are visible.  I see `sstore`, `codecopy`, `mstore8`, `div`, `exp` and so on.  So I don't need this test.